lib/py/src/TRecursive.py

```diff
@@ -0,0 +1,63 @@
+# MODIFIED June 20, 2017, Eric Conner
+#
+#
+# Original source copyright 2014-present Facebook, Inc.
+#
```

Comment:
Where does line 1 to 5 come from? That's not part of the standard ASF license header.